server/engine/src/main/antlr4/org/eclipse/lsp/db2/parser/Db2SqlParser.g4

```diff
@@ -64,6 +68,10 @@ xml_lobNO_size: BLOB_FILE | CLOB_FILE | DBCLOB_FILE;
 
 xml_as: XML AS;
 
+rowid: ROWID;
```

Comment:
we could use `ROWID ` directly at `rowid_host_variables_arrays`, instead of creating a rule for it